### DISCUSSION TITLE
npm install issue

### DISCUSSION THREAD

**Discussion by Pawanupadhyay10** [polypheny/Polypheny-DB#319] [Q&A] [answered]
after running npm and then according to instructions ,after running  ./gradlew npm_install ,then next  http://localhost:4200/. is not working.
can u guide me what i did wrong.

**Comment by vogti** [comment, marked as answer]
You did nothing wrong. The documentation is not good at this point. The gradle command does not start the UI. You need to install NodeJS on your machine. If you have then that, follow these steps:
- Execute `npm install` install in the root folder of the Polypheny-UI repository. 
- Install Angular: `npm install -g @angular/cli`
- `ng serve --aot -o`

You should now be able to open the UI in your browser (http://localhost:4200/).

**Comment by Pawanupadhyay10** [comment]
Thanks it setup.
